build.ps1

```diff
@@ -306,7 +306,13 @@ $MSBuildArguments += "/p:TargetOsName=win"
 if (($All -or $BuildJava) -and -not $NoBuildJava) {
     $foundJdk = $false
     $javac = Get-Command javac -ErrorAction Ignore -CommandType Application
-    if ($env:JAVA_HOME) {
+    $localJdkPath = "$PSScriptRoot\.tools\jdk\win-x64\"
```

Comment:
That's an interesting solution. We'd want a `clean.cmd` script to go with this that doesn't delete `.dotnet` or `.tools` (or the SignalR js client 😆).

Reply:
Could also be a simple git command.
```
git config --global alias.cleanme 'clean -xfd -e .dotnet -e .tools -e src/SignalR/clients/ts/FunctionalTests/node_modules/'
```

Reply:
"simple" 😁 that's why I suggested a cmd, this affects the whole team.

Reply:
Clean makes me a little nervous. Can we discuss this one separately? This is the first request for a clean command I've heard. As clean is potentially destructive, I want to make sure we capture the right intent here so we don't end up with a script that deletes your unstaged code.

Reply:
https://github.com/aspnet/AspNetCore-Internal/issues/2532